packages/dso-toolkit/src/global/mixins/setcolors.mixin.scss

```diff
@@ -0,0 +1,45 @@
+@use "../../variables/colors.scss";
+
+@mixin setColors($bgcolor: null) {
```

Comment:
Ik voorzag deze mixin als ondersteunend in plaats van leidend. Door in deze mixin een kleur mapping te plaatsen ontnemen wij onszelf de vrijheid in de componenten. Verdwijnt met deze opzet ook variablen zoals `alert.$success-bg` ?

Ik zat eerst hieraan te denken:

```scss
@mixin set-colors($background-color, $color, $link-color, $icon-color)
```

Maar tijdens het typen bedenk ik mij dat we ook wel zonder `$color` en `$link-color` kunnen, die staan immers vast en het patroon is helder: Op achtergrond kleur zwarte linkkleur.

Dus misschien dat we wel gewoon naar dit kunnen:

```scss
@mixin background-color($background-color)
```

Als `$background-color == $white`, zetten we de standaard linkkleur en zwarte tekstkleur. Anders zetten we de zwarte linkkleur en tekstkleur.

Wat denk jij? @anneke kan jij ook meedenken?

Reply:
Volgens mij zijn we dan weer zo goed als terug bij de oorspronkelijke opzet: allen de achtergrondkleur, de rest is geautomatiseerd. Over de arguments kunnen we het zeker nog hebben ;)

groeten uit Parijs! (Op m’n rug op bed, zo weer weg richting diner)

Reply:
Heb net 2 x 3 uur in de Thalys gezeten, dus dat treintje kan er ook nog wel bij 😁